erizo/src/erizo/media/ExternalOutput.h

```diff
@@ -82,7 +83,7 @@ class ExternalOutput : public MediaSink, public RawDataReceiver, public Feedback
   // and that value is added to every timestamp we write.
   long long firstVideoTimestamp_;  // NOLINT
   long long firstAudioTimestamp_;  // NOLINT
-  long long firstDataReceived_;  // NOLINT
+  clock::time_point firstDataReceived_;  // NOLINT
```

Comment:
This was a good opportunity to ditch CamelCase in the variables involved in this PR.